### ISSUE TITLE
Reword guidelines for lithostratigraphy code tables

### ISSUE THREAD

**Issue by Nicole-Ridgwell-NMMNHS** [ArctosDB/arctos#9247]
**Group**

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlithostratigraphic_group

Controlled vocabulary for lithostratigraphic Groups and Supergroups, available as locality attributes. Terms may be added to this table without DBA approval and with the approval of one active Code Table Committee member, as long as that Code Table member did not also request the term if the following criteria are met:

The unit is registered in a national geologic database, with a link included in the documentation.
https://ngmdb.usgs.gov/Geolex/search
https://www.bgs.ac.uk/technologies/the-bgs-lexicon-of-named-rock-units/
If the unit is not listed in a national database, the documentation must include a link to a primary literature publication that describes the unit.

Terms must be consistently formatted. Acceptable examples include:

Dunquin Group

Old Red Sandstone Supergroup

In addition to the required links, the documentation must include a short description of the unit which includes

the geographic region(s) in which the unit outcrops,
geologic age
dominant lithology
Write for an international audience; do not abbreviate.

**Formation**

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlithostratigraphic_formation

Controlled vocabulary for lithostratigraphic formations available as locality attributes. Terms may be added to this table without DBA approval and with the approval of one active Code Table Committee member, as long as that Code Table member did not also request the term if the following criteria are met:

The unit is registered in a national geologic database, with a link included in the documentation.
USGS National Geologic Map Database (https://ngmdb.usgs.gov/Geolex/search)
BGS Lexicon of Named Rock Units (https://www.bgs.ac.uk/technologies/the-bgs-lexicon-of-named-rock-units/)
Australian Stratigraphic Units Database (https://asud.ga.gov.au/search-stratigraphic-units/)
If the unit is not listed in a national database, the documentation should include a link to a publication in the primary literature that describes the unit.
The term is properly formatted as "Name Type". Lithology descriptors should only be included when the unit is not referred to both with and without the descriptor. When a lithology descriptor is included, the format is "Name Lithology Type". Acceptable examples:
Cierbo Formation
Clare Shale Formation
In addition to the required links, the documentation must include a short description of the unit which includes
the geographic region(s) in which the unit outcrops,
geologic age
dominant lithology
Write for an international audience; do not abbreviate.

**Member**

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlithostratigraphic_member

Controlled vocabulary for lithostratigraphic members available as locality attributes. Terms may be added to this table without DBA approval and with the approval of one active Code Table Committee member, as long as that Code Table member did not also request the term if the following criteria are met:

The unit is registered in a national geologic database, with a link included in the documentation.
https://ngmdb.usgs.gov/Geolex/search
https://www.bgs.ac.uk/technologies/the-bgs-lexicon-of-named-rock-units/
If the unit is not listed in a national database, the documentation should include a link to a publication that describes the unit.
The term is properly formatted as "Name Type". Lithology descriptors should only be included when the unit is not referred to both with and without the descriptor. When a lithology descriptor is included, the format is "Name Lithology Type". Acceptable examples: Cameron Creek Member, Down Cliff Sand Member
In addition to the required links, it is strongly preferred that the documentation include a short description of the unit. Information suggested to be in the description includes: the geographic region(s) in which the unit outcrops, geologic age, and dominant lithology.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
@ArctosDB/paleontology

**Comment by dustymc** [comment]
General request: If we end up with overlapping "authority" (reference? The bar is still a little fuzzy from here), please indicate which is preferable/rank them.

**Comment by WaigePilson** [comment]
> Controlled vocabulary for lithostratigraphic formations available as locality attributes.

I'd propose we reword the documentation to say:
Controlled vocabulary for lithostratigraphic formations available as locality attributes. Terms may be added to this table without DBA approval and with the approval of one active Code Table Committee member, as long as that Code Table member did not also request the term if the following criteria are met:

The unit is registered in a national geologic database, with a link included in the documentation.
- National geologic map authority such as USGS National Geologic Map Database (https://ngmdb.usgs.gov/Geolex/search), Australian Stratigraphic Units Database (https://asud.ga.gov.au/search-stratigraphic-units/), WEBLEX Canada (https://weblex.canada.ca/weblexnet4/weblex_e.aspx), BGS Lexicon of Named Rock Units (https://www.bgs.ac.uk/technologies/the-bgs-lexicon-of-named-rock-units/)
- or (less preferred) macrostrat (https://macrostrat.org/)

If the unit is not listed in a national database, the documentation should include a link to a publication in the primary literature that describes the unit.
The term is properly formatted as "Name Type". Lithology descriptors should only be included when the unit is not referred to both with and without the descriptor. When a lithology descriptor is included, the format is "Name Lithology Type". Acceptable examples:
Cierbo Formation
Clare Shale Formation
In addition to the required links, the documentation must include a short description of the unit which includes
- the geographic region(s) in which the unit outcrops,
- geologic age
- dominant lithology

Write for an international audience; do not abbreviate.

**Comment by WaigePilson** [comment]
^and I'd suggest we use the same wording for lithostratigraphic_bed, lithostratigraphic_member, and lithostratigraphic_group.

Question for the whole group: do we want to accept wikipedia as a valid reference for metadata? I heard at least one person endorse this!

**Comment by dustymc** [comment]
>say:...Controlled vocabulary 

Can I vote to get rid of that? I think I heard we want something more authoritative than just a dictionary, which this implies.


> or (less preferred) macrostrat (https://macrostrat.org/) .... If the unit is not listed in a national database, ....... wikipedia

AHA!, maybe. I think possibly there are two things going on here.

1. If some criteria (eg national database) are met, anyone on the CT committee can review for obvious typos, check links, etc., and add things. I think this has worked out well so far, but I'm not sure I can do this if eg wikipedia were involved - someone requests whatever https://en.wikipedia.org/wiki/Special:Random lands on, and now instead of checking for typos I have to determine if https://en.wikipedia.org/wiki/Vitorino_Hilton is actually a formation or not.
2. If those criteria aren't met, the request needs review by someone who understands more than I do (and an appropriate committee now exists!). In that situation, I don't care about the standardized references, and the issue/discussion will probably be the important thing. (And surely that committee will notice if the unit is actually in one of those DBs - those references are important to things like duplicate detection.) 

If I'm not lost, possibly we want to keep the "streamline references" (eg stuff I can use) about like they are, and clarify that The Committee can reference less-focused resources as necessary??

**Comment by WaigePilson** [comment]
> If some criteria (eg national database) are met, anyone on the CT committee can review for obvious typos, check links, etc., and add things. I think this has worked out well so far, but I'm not sure I can do this if eg wikipedia were involved...

If you're checking whether the geolex (or any other national database) matches, aren't you also checking if it's a formation or not? For instance, https://ngmdb.usgs.gov/Geolex/Units/MariasRiver_9180.html doesn't explicitly say what level of geologic unit Marias River is, but we added it as the Marias River Formation to the lithostratigraphic_formation code table based on this geolex entry and our knowledge of the unit. Wikipedia, however, clearly calls it a formation https://en.wikipedia.org/wiki/Marias_River_Shale

My point just being that wikipedia is does not necessarily require more work or expertise from the CT committee. Any of these sources we are using (national databases, macrostrat, wikipedia, primary literature) are not a definitive authority, and there will always need to be some vetting from whoever vets the code table submission or trust that the submitter has done their due diligence in creating the code table request.

**Comment by dustymc** [comment]
>we added it as the Marias River Formation 

No, we did what the documentation said not to do...

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlithostratigraphic_formation#marias_river_shale_formation

(And it wasn't me, check the log!)

ANYWAY - yea, I'm going to check that the links works and for obvious typos and Excel-shuffles and such, and definitely rely on ya'll to not request nonsense, and maybe that does mean I could work with wikipedia, and maybe that should be formalized somehow - "when requested by active members of The Paleo Committee...." ??? I'm up for trying about anything, just hoping to find some path that future-us won't regret! For reference, I'm about positive we'd be time ahead by now - and have better data - if we'd have spent a couple weeks cleaning up a bunch of stuff that got bulkloaded without references - but of course we didn't know that then....

**Comment by Jegelewicz** [comment]
Why not have the paleo committee vet these things? These are very specific to a community and it seems like leaving them to the code table committee is causing slow downs. If the paleo committee approves, then add the term (as it is a committee, I would assume several sets of eyes and brains would review to eliminate typos, etc).

**Comment by dustymc** [comment]
Here's a stab at formation, which I hope can serve as a template for other litho-stuff.

I've removed some misleading and incorrect terminology (these are much more than controlled vocabularies!).

I've tried to separate expectations from procedures.

I've tried to simplify and clarify the "fast add" pathway.

I've tried to clarify what's necessary and nice when 'fast add' data don't exist.

I changed the examples to include something that needs a "lithology descriptor," but I'm not sure that's clear yet.

I included the proposal from https://github.com/ArctosDB/arctos/issues/9160 - so please reject it here if we don't want it for some crazy reason.

My current view of this is basically

* if it's in one of the national DBs I (et al.) can just add it, or
* if it's not then The Committee needs to review, and would be expected to include some sort of evidence that the term is actually a "formal lithostratigraphic formation"


Please fix whatever of that's wrong, esp. if there's some fundamental change regarding macrostrat (I've not been entirely clear on that).

HELP!

-------------

"Formal" lithostratigraphic formations; Data should follow a national geologic database or be attributable to a recognized type publication.

Terms should include the unit type (generally "Formation") Lithology descriptors should only be included if there are multiple valid units with similar names where the lithology term is the distinction


*  Duck Creek Formation 
*  Duck Creek Limestone Formation 

Description should include a concise summary of the unit, including:

* geographic region(s) in which the unit outcrops
* geologic age of the unit
* (optionally) dominant lithology


Write for an international audience; **do not** abbreviate.


Include clarifying links (such as Wikipedia) when available.


Terms which occur in the following databases may be added to this table after review by one active Code Table Committee member, as long as that Code Table member did not also request the term.

* USGS National Geologic Map Database (https://ngmdb.usgs.gov/Geolex/search)
* BGS Lexicon of Named Rock Units (https://www.bgs.ac.uk/technologies/the-bgs-lexicon-of-named-rock-units/)
* Australian Stratigraphic Units Database (https://asud.ga.gov.au/search-stratigraphic-units/)
* WEBLEX Canada (https://weblex.canada.ca)


The Arctos Paleo Committee must approve any additions which do not meet the above criteria. These entries will require additional descriptive data and references, and should cite primary literature type publications when possible.


------------

**Comment by WaigePilson** [comment]
> Documentation should include a concise summary of the unit, including:

Subcommittee suggests update to:
Description should include a concise summary of the unit, including:
- geographic region(s) in which the unit outcrops
- geologic age of the unit
- (optionally) dominant lithology

Also, clarifying that any code table addition submitted without one of the listed references would need additional vetting (more than the approval of one DBA)

**Comment by WaigePilson** [comment]
> Terms should include the unit type (generally "Formation.") Lithology descriptors should only be included when the unit is referred to both with and without the descriptor, for example:

Subcommittee suggests: "Terms should include the unit type (generally "Formation") Lithology descriptors should only be included if there are multiple valid units with similar names where the lithology term is the distinction"

**Comment by KatherineLAnderson** [comment]
I can take a look at the remaining formations that Paige didn't already vet.

**Comment by dustymc** [comment]
>concise summary...

>>"Terms should include ...

edited

>clarifying


If


>The Arctos Paleo Committee must approve any additions which do not meet the above criteria. These entries will require additional descriptive data and references, and should cite primary literature type publications when possible.

isn't clear enough, please reopen with specifics.

Otherwise done!
